src/views/news/NewsView/NewsView.vue

```diff
@@ -48,7 +48,7 @@
           filters: route.query.filters && (route.query.filters as string).split(','),
         },
       })
-    ).data.data as NewsData;
+    ).data.data as NewsItem;
```

Comment:
This creates a lot of type errors in the template

Reply:
can be addressed in https://github.com/eternagame/eternagame.org/issues/134